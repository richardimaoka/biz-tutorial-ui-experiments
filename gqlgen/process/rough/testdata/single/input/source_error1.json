{
  "phase": "サーバーを動かす",
  "type": "source error"
}
