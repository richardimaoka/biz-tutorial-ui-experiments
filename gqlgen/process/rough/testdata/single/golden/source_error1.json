[
  {
    "fromRoughStep": true,
    "subId": "sourceErrorStep",
    "step": "",
    "focusColumn": "Source Code",
    "column1": "Source Code"
  }
]